src/main/java/dev/jbang/source/KotlinScriptSource.java

```diff
@@ -0,0 +1,45 @@
+package dev.jbang.source;
```

Comment:
I'm not per se against introducing subclasses of `ScriptSource`, but we should then take it to its logical conclusion and introduce `JavaScriptSource` as well.

Reply:
it doesn't work that great though as java is also .jsh....so this should probably be handled differently.

Reply:
leaving this in for now - should be refactored to not need/want a subclass